tests/Feature/AppointmentTest.php

```diff
@@ -33,9 +33,12 @@ public function testGetAllAppointments(): void
 
     public function testCreateAppointment():void
     {
+        $user = User::factory()->create();
+        $type = Type::factory()->create();
+
         $data = [
-            'user_id' => 1,
-            'type_id' => 5,
+            'user_id' => $user->id,
```

Comment:
NOTE: This change was added based on a failing test from this CI process 🎉 
Turns out I had not created a new user or type and locally it wasn't failing because my database has seeded data.